app/models/npc.rb

```diff
@@ -0,0 +1,51 @@
+class Npc < ActiveRecord::Base
+  class ConsequenceLimit < ActiveModel::Validator
+    def validate(npc)
+      if npc.consequences.length > 4
```

Comment:
That could be the case in some code bases by convention but in practice Ruby treats any token beginning with an uppercase letter as a constant. If you go into IRB and define one and then re-assign it, you should get a warning but you can do it. I think for Enum-style types that natural casing reads better in use but you can use any format you like. :wink: